src/pandas_profiling/report/structure/variables/render_timeseries.py

```diff
@@ -239,11 +235,22 @@ def render_timeseries(config: Settings, summary: dict) -> dict:
         sequence_type="grid",
     )
 
+    if isinstance(summary["histogram"], list):
+        hist_data = histogram(
+            config,
+            [x[0] for x in summary["histogram"]],
+            [x[1] for x in summary["histogram"]],
```

Comment:
If `summary["histogram"]` has 2-tuples,
```suggestion
            *zip(*summary["histogram"]),
```
should do by my reckoning.